src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -98,12 +98,14 @@ interface ItemListLayoutUserSettings {
 @observer
 export class ItemListLayout extends React.Component<ItemListLayoutProps> {
   static defaultProps = defaultProps as object;
+
+  private watchDisposers: IReactionDisposer[] = [];
+
   @observable hiddenColumnNames = new Set<string>();
   @observable isUnmounting = false;
 
-  // default user settings (ui show-hide tweaks mostly)
   @observable userSettings: ItemListLayoutUserSettings = {
-    showAppliedFilters: false,
+    showAppliedFilters: true,
```

Comment:
Should be `false` by default.

Reply:
I put it to `true` since it might be confusing now for end users because not all resources loaded and you see something like: **Filtered: 30/30** where numbers are mostly always the same (unless you change the search filter).

I remember even one issue about "how to reset all namespaces". So with this thing open by default it's more clear imho.

Reply:
Let's not change this behaviour in this PR 🙂

Reply:
Alright! Should I create another PR for that?